src/main/kotlin/app/ui/userCreation/View/UserCreation.kt

```diff
@@ -0,0 +1,157 @@
+package app.ui.userCreation
+
+import app.MyApp.Companion.Colors
+import app.ui.styles.ButtonStyles
+import app.ui.userCreation.ViewModel.UserCreationViewModel
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Pos
+import javafx.scene.effect.DropShadow
+import javafx.scene.paint.Color
+import tornadofx.*
+import app.widgets.recordButton.RecordButton
+import app.ui.welcomeScreen.*
+import javafx.application.Platform
+import java.util.*
+import kotlin.concurrent.timerTask
+import app.ui.ProgressBar
+import io.reactivex.rxkotlin.subscribeBy
+import app.ui.profilePreview.View.ProfilePreview
+import javafx.scene.layout.VBox
+
+/*
+*  This View is used in the process of creating user profiles
+*  This View (UserCreation) Which serves as the parent view
+*  has two children. A close button, and a recordingFrame/recordButton that
+*  gives the user visual feedback through the recording process
+*
+* */
+class UserCreation : View() {
+
+
+    private val closeIcon = MaterialIconView(MaterialIcon.CLOSE, "25px")
+    private val viewModel: UserCreationViewModel  by inject()
+    var recordButton = RecordButton(::onClickCallback, ::animationCompleted, ::stopClicked)
+    private val progressBar = ProgressBar()
+    val isRecording = viewModel.isRecording
+    val doneRecording = viewModel.doneRecording
+    var timer = Timer()
+    val RECORDING_DONE: Long = 6100
+
+    //initialize close button to be used in top of borderpane
+    val closeButton = button(messages["close"], closeIcon) {
+        importStylesheet(ButtonStyles::class)
+        addClass(ButtonStyles.rectangleButtonDefault)
+
+        style {
+            alignment = Pos.CENTER
+            closeIcon.fill = c(Colors["accent"])
+            effect = DropShadow(10.0, Color.GRAY)
+        }
+        action {
+            navHome()
+        }
+    }
+
+    override val root = borderpane {
+
+        style {
+            backgroundColor += Color.WHITE
+        }
+
+        top {
+
+            //close button
+            hbox {
+                alignment = Pos.BOTTOM_RIGHT
+                add(closeButton)
+                style {
+                    alignment = Pos.BOTTOM_RIGHT
+                    paddingRight = 40.0
+                    paddingTop = 40.0
+                }
+            }
+
+        }
+        center {
+
+            isRecording.subscribeBy(
+                    onNext = {
+                        if (it == false) { // if isRecording set to false create new RecordButton()
+                            val newRecordButton = RecordButton(::onClickCallback, ::animationCompleted, ::stopClicked)
+                            newRecordButton.alignment = Pos.CENTER
+                            recordButton.replaceWith(newRecordButton)
+                            recordButton = newRecordButton
+                        }
+                    },
+
```

Comment:
don't have a new line in a parameter list

Reply:
I'm unsure what you mean by this